packages/react-components/style/output-components-style.css

```diff
@@ -93,27 +97,80 @@ canvas {
     align-self: center;
 }
 
-.table-tree>tbody>tr:nth-child(1)>th {
-    background-color: var(--theia-editor-background);
-    position: sticky;
-    top: 0;
+.table-tree {
+    display: grid;
+    border-collapse: collapse;
+    border-left: 1px solid #CCC;
```

Comment:
#CCC is to bright when in dark theme. We should use a themed color of theia. I could find one that is meant for tree borders,  but var(--theia-tree-inactiveIndentGuidesStroke) seems to be a good one. Also add a right border which looks good to have on the right.
```
     border-left: 1px solid var(--theia-tree-inactiveIndentGuidesStroke) !important;
     border-right: 1px solid var(--theia-tree-inactiveIndentGuidesStroke) !important;
```

Reply:
I got a weird result adding the border-right to the table. The table border was not moved when resizing the columns.

The border-right is provided by the td elements. I had disabled it on the filler column. I removed that and forced it on the filler elements also, which includes the th filler, since th don't have that right border because of the resize handle.

Note that I made cells have bottom and right border only because border-collapse didn't seem to work right (so there was double-border between cells). Maybe that could have been fixed some other way.